docs/content/environment_creation.md

```diff
@@ -68,8 +68,8 @@ from pettingzoo.utils.deprecated_module import DeprecatedModule
 knights_archers_zombies_v0 = DeprecatedModule("knights_archers_zombies", "v0", "v10")
 ```
 This declaration tells the user that `knights_archers_zombies_v0` is deprecated and `knights_archers_zombies_v10` should be used instead. In particular, it gives the following error:
-``` python
+``` python notest
 from pettingzoo.butterfly import knights_archers_zombies_v0
 knights_archers_zombies_v0.env()
-# pettingzoo.utils.deprecated_module.DeprecatedEnv: knights_archers_zombies_v0 is now deprecated, use knights_archers_zombies_v10 instead
+
```

Comment:
I think the deprecated module code above is fine to keep just notest it, or maybe it’s possible to do a doc test type thing with the
>>> and it checks that the error message is what you’d expect? Not sure. I guess that’s more for docstring in Python files rather than docs files. Fine to not test this if you can’t find a simple way

Reply:
Actually I think I deleted this comment here by mistake, I'm gonna just add it back and keep the `notest`